gap/projective/AnSnOnFDPM.gi

```diff
@@ -58,7 +58,7 @@ end;
 
 ## Functions for efficient matrix operations
 
-RECOG.Commute := function(g, h);
+RECOG.AreCommutingNonProjectiveMatrices := function(g, h);
```

Comment:
Matrices are by default not projective, so this feels overly redundant.
```suggestion
RECOG.AreCommutingMatrices := function(g, h);
```
Also, one immediately wonders why this doesn't use your new AreCommuting/DoCommute methods. Presumably there is a reason -- if so, it should be stated in a comment. (My guess is: you are concerned this is called with "projective" groups)

Finally, why change this but not the Conj and ConjInv methods afterwards?

Reply:
I agree. It is unclear why we shouldn't use the new `AreCommuting/DoCommute` method. I thought the idea was to use this the same way as the `isone` entry in the recog node in order to deal with projective groups.